internal/commands/login.go

```diff
@@ -48,9 +49,14 @@ func newLoginCmd(streams command.Streams, store credentials.Store, hubClient *hu
 			if len(args) > 0 {
 				username = args[0]
 			}
-			if err := login.RunLogin(cmd.Context(), streams, hubClient, store, username); err != nil {
+			err := login.RunLogin(cmd.Context(), streams, hubClient, store, username)
+			if err != nil {
+				if err == errdef.ErrCanceled {
```

Comment:
nit: I guess we should check the error using [errors.Is](https://golang.org/pkg/errors/#Is)